tests/run/cpp_class_attrib.srctree

```diff
@@ -0,0 +1,78 @@
+PYTHON setup.py build_ext --inplace
+PYTHON -c "import runner"
+
+######## setup.py ########
+
+from Cython.Build.Dependencies import cythonize
+from distutils.core import setup
+import os
+
+examples = '../../../docs/examples/userguide/wrapping_CPlusPlus'
```

Comment:
I would prefer having a simple version of the Rectangle C++ source here, rather than looking it up in some far away directory through brittle relative paths. You can use [verbatim C code](http://docs.cython.org/en/latest/src/userguide/external_C_code.html#including-verbatim-c-code) for that.

Also, I'm currently enabling the C++ tests on appveyor (#2797). Once that works and I merged it, you can merge the latest master into your feature branch to let appveyor run this test.

Reply:
Whoops, I didn't mean for the cpp attribute example to creep in. The feature I am adding is only for `ctypedef`-defined classes.